src/plugins/intel_gna/src/frontend/scale_factor_calc.cpp

```diff
@@ -1238,8 +1239,9 @@ bool ScaleFactorCalculator::ScaleFactorPerLayerWeightable(InferenceEngine::Weigh
         }
         quant->_weights_quant.SetScale(quant->_weights_quant.GetScale() / weights_reducer);
     }
+
     double tmp_dst_quant_scale = quant->_weights_quant.GetScale() * quant->_src_quant.GetScale();
-    if (weights_prec == InferenceEngine::Precision::I8) {
+    if (weights_prec == InferenceEngine::Precision::I8 && !(LayerInfo(wl).isConvolution() || LayerInfo(wl).isConvolutionFilter())) {
```

Comment:
isn't it compound bias as well?
```suggestion
    if (is_bias_compound) {
```

Reply:
Yes, it is, but I wanted to have convolution here mentioned explicitly to allow easier refactoring. The plan is to try to split convolutions from other weightable layers. Besides there is no mention of MAX_OUT_MULTIPLIER here, so I think it's easier to read this way.